homeassistant/components/buienradar/sensor.py

```diff
@@ -197,13 +211,15 @@
         name="Rain last 24h",
         native_unit_of_measurement=LENGTH_MILLIMETERS,
         icon="mdi:weather-pouring",
+        state_class=STATE_CLASS_TOTAL_INCREASING,
     ),
     # new in json api (>1.0.0):
     SensorEntityDescription(
         key="rainlasthour",
         name="Rain last hour",
         native_unit_of_measurement=LENGTH_MILLIMETERS,
         icon="mdi:weather-pouring",
+        state_class=STATE_CLASS_TOTAL_INCREASING,
```

Comment:
Setting this to `total_increasing` means we'll be collecting a statistics of lifetime rainfall, is that wanted?

Reply:
True, but only if the sensor is manually enabled from the entities screen ([they are disabled by default](https://www.home-assistant.io/integrations/buienradar/#sensor)). 
For more control the user can set the `max_age` on the statistic.

Reply:
Setting `max_age` on the sensor means it will reset to 0?
That is very unlikely to work well with state class set to `STATE_CLASS_TOTAL_INCREASING`, or have you tested it?

Reply:
I haven't tested it, but that's what the [documentation](https://www.home-assistant.io/integrations/statistics/#max_age) says.

Reply:
I would set it to `STATE_CLASS_MEASUREMENT`, same as the precipation sensor. It's the sams type of sensor but instead of future it is historic data.

Reply:
Wouldn't that make the measurements invalid?

That's why I made it an increasing measurement, also see https://developers.home-assistant.io/docs/core/entity/sensor/#state-class-total_increasing

Reply:
Hmm...I know what you mean. The value reported is an accumulated value over the last 24 hours. But total increasing would also give invalid measurements, because the total increasing would only work when e.g. at 0:00 a counter starts at zero starting counting up to 23:59 and at 0:00 it would reset again.

Buienradar reports an accumulated value over the last 24h. The fact that it goes down doesn't mean a new cycle starts...
I don't think there is a state class which would fit to this, because it's (I think) impossible to derive statistics from this value.

Reply:
Although measurement would still be better than total increasing I guess.

Reply:
True, I think both will result in invalid measurements. I removed the `state_class` from both "Rain last hour" and "Rain last 24h" since it won't be accurate anyway.